TopPlaces/TopPlaces/PhotosHistory.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+@import Foundation;
```

Comment:
We use <code>#import</code>. 
You can add common imports to the precompiled header usually named ProjectName-Prefix.pch which is missing in your project for some reason.